assignments/objects.js

```diff
@@ -19,26 +19,76 @@ const example = {
 
 // Write your intern objects here:
 
+const internMit = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0psu.edu",
+  "gender": "F"
+}
+
+const internKen = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": () => {
+    console.log("Hello, my name is Kennan!");
```

Comment:
You should make this a return, instead of console log. This gets the answer you're looking for at line 88.

```suggestion
    return "Hello, my name is Kennan!";
```